stable/datadog/templates/system-probe-init.yaml

```diff
@@ -1,6 +1,10 @@
 {{- define "system-probe-init" -}}
 - name: seccomp-setup
+  {{- if semverCompare "^6.15.x" .Values.image.tag }}
   image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
Why can't this simply be `.Values.image.tag`?

Reply:
In case `image.tag` isn't at least 6.15. The system probe init container must run with at least 6.15.

I honestly found it a bit odd to allow enabling the system probe containers at all without the main image being at least 6.15. But that logic is what was here before so I maintained it.

Reply:
That said, I'm totally open to adjusting as needed.

Reply:
It is fine to remove the logic around 6.15. It was a mistake to have this value hardcoded.